docs/src/pages/components/dialogs/ConfirmationDialog.js

```diff
@@ -137,8 +137,8 @@ export default function ConfirmationDialog() {
 
   return (
     <div className={classes.root}>
-      <List component="div" role="list">
-        <ListItem button divider disabled role="listitem">
+      <List component="div" role="group">
```

Comment:
Makes more sense to group them with the current implementation of List (related to https://github.com/mui-org/material-ui/issues/13597#issuecomment-642518895). I'd question the demo as a whole but this requires more context. It's using List for styles and has to go the extra mile to remove the implied semantics.

Let's leave it as is to get the build green and revisit this when working on #13597

Reply:
@dependabot squash and merge